web/src/components/molecules/WorkspaceSettings/Form.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback } from "react";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Form from "@reearth-cms/components/atoms/Form";
+import Input from "@reearth-cms/components/atoms/Input";
+import { useT } from "@reearth-cms/i18n";
+
+export type Props = {
+  workspaceName?: string;
+  onWorkspaceUpdate: (name?: string | undefined) => Promise<void>;
+};
+
+const WorkspaceForm: React.FC<Props> = ({ workspaceName, onWorkspaceUpdate }) => {
+  const [form] = Form.useForm();
+  const t = useT();
+
+  const handleSubmit = useCallback(async () => {
+    try {
+      const values = await form.validateFields();
+      await onWorkspaceUpdate?.(values.name);
+    } catch (info) {
+      console.log("Validate Failed:", info);
```

Comment:
Can you handle this as a notification? Or if its not necessary because antd's form will handle the feedback, remove the try/catch

Reply:
@KaWaite , no need for `try/catch` after I've added notification to hooks/error handler.